src/NotFoundPage.test.jsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react';
+
+import NotFoundPage from './NotFoundPage';
+
+describe('NotFoundPage', () => {
+  it('renders not found page', () => {
+    const { container } = render(
+      <NotFoundPage />,
+    );
```

Comment:
```suggestion
    const { container } = render((
      <NotFoundPage />
    ));
```

괄호를 두번 감싸주시면 콤마를 없앨 수 있어요.